CONTRIBUTING-detailed-instruction.md

```diff
@@ -0,0 +1,100 @@
+## Detailed Instructions
+### Setup
+
+1. You need a [GitHub account](https://github.com)
+2. [Install](https://help.github.com/articles/set-up-git/#platform-all) `git` on your computer.
+3. **`Fork`** the scalabridge/curriculum repository:
+	1. On GitHub, navigate to the [scalabridge/curriculum](https://github.com/scalabridge/curriculum) repository.
+	2. In the top-right corner of the page click **Fork**.
+3. **Create a local `clone`** of your fork to put the repository on your local computer:
+	1. On GitHub, navigate to **your fork** of the scalabridge/curriculum repository.
+	2. Under your repository name, to the right, click **Clone or download**.
+	3. In the **Clone with HTTPs** section, click the "Copy to clipboard" icon to copy the clone URL for the repository.
+	4. On Linux or Mac, open a `terminal`, on Windows open `Git Bash`. From the prompt, if needed, change the working directory to be the one in which you want the repository files created.
+	5. Type `git clone `, and then paste the URL you copied in step 3 above. The command line should look like this, with your GitHub username instead of YOUR-USERNAME:
+	```bash
+	$ git clone https://github.com/YOUR-USERNAME/curriculum.git
+
+	Press `Enter` and git will create your local clone
```

Comment:
The formatting is a bit tricky to get right -- here's one way to do it (make sure you match the indentation and blank lines exactly):

```
3. **Create a local `clone`** of your fork to put the repository on your local computer:
    1. On GitHub, navigate to **your fork** of the scalabridge/curriculum repository.
    2. etc...
    5. Type `git clone `, and then paste the URL you copied in step 3 above. The command line should look like this, with your GitHub username instead of YOUR-USERNAME:

        ```bash
        $ git clone https://github.com/YOUR-USERNAME/curriculum.git
        ```
    Press `Enter` and git will create your local clone
5. Link the Github (remote) version of the repository with your local version so it will synchronize before you make changes:
```